django_mongodb/schema.py

```diff
@@ -9,13 +9,26 @@ def delete_model(self, model):
         self.connection.database[model._meta.db_table].drop()
 
     def add_field(self, model, field):
-        pass
+        # Create implicit M2M tables
+        if field.many_to_many and field.remote_field.through._meta.auto_created:
+            self.create_model(field.remote_field.through)
+            return
+        default = self.effective_default(field)
```

Comment:
This variable is only used in the line 18, could we move this line or erase it? or the method must be called?